apps/bfd-db-migrator/src/main/java/gov/cms/bfd/migrator/app/DatabaseOptions.java

```diff
@@ -0,0 +1,65 @@
+package gov.cms.bfd.migrator.app;
+
+import java.io.Serializable;
+
+/** The user-configurable options that specify how to access the application's database. */
+public final class DatabaseOptions implements Serializable {
+  private static final long serialVersionUID = 1L;
```

Comment:
does this need serializable?

Reply:
I assume this was because AppConfiguration was, but i'll remove it until I see why it would need to be.